src/standalone/Snow/Snow.jl

```diff
@@ -2,11 +2,48 @@ module Snow
 
 using DocStringExtensions
 import ...Parameters as LP
-export SnowParameters
+using ClimaCore
+using Thermodynamics
+using ClimaLand
+using ClimaLand:
+    AbstractAtmosphericDrivers,
+    AbstractRadiativeDrivers,
+    turbulent_fluxes,
+    net_radiation,
+    AbstractModel,
+    heaviside
+
+import ClimaLand:
+    make_update_aux,
+    make_compute_exp_tendency,
+    prognostic_vars,
+    auxiliary_vars,
+    name,
+    prognostic_types,
+    prognostic_domain_names,
+    auxiliary_types,
+    auxiliary_domain_names,
+    surface_temperature,
+    surface_specific_humidity,
+    surface_height,
+    surface_albedo,
+    surface_emissivity,
+    get_drivers
+export SnowParameters, SnowModel
+
+struct AbstractSnowModel{FT} <: AbstractModel{FT} end
```

Comment:
I wonder (open question) if we should add a short docstring here too

Reply:
I added something in!